js/scripts.js

```diff
@@ -0,0 +1,4 @@
+var text = 'Velociraptor is a genus of herbivorous ceratopsid dinosaur that first appeared during the late Maastrichtian stage of the late Cretaceous period.';
+var dinosaur = 'triceratops'.toUpperCase();
+var textCharsAfter = text.replace('Velociraptor' , dinosaur);
```

Comment:
Zrób tak żeby zamiast Velociraptor, zamieniany był zawsze pierwszy wyraz zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody `split` (google -> "mdn split")